outputs.tf

```diff
@@ -1,23 +1,45 @@
 output "fargate_service" {
-  value = module.fargate.ecs_service
+  value = aws_ecs_service.service
 }
 
 output "fargate_service_security_group" {
-  value = module.fargate.service_sg
+  value = aws_security_group.fargate_service_sg
 }
 
-output "codedeploy_appspec_json" {
-  value = module.fargate.codedeploy_appspec_json
+output "task_definition" {
+  value = aws_ecs_task_definition.task_def
+}
+
+output "codedeploy_deployment_group" {
+  value = aws_codedeploy_deployment_group.deploymentgroup
+}
+
+output "codedeploy_basic_appspec_json" {
+  value = jsonencode({
+    version = 1
+    Resources = [{
+      TargetService = {
+        Type = "AWS::ECS::SERVICE"
+        Properties = {
+          TaskDefinition = aws_ecs_task_definition.task_def.arn
+          LoadBalancerInfo = {
+            ContainerName = local.container_definitions[0].name
```

Comment:
It seems interesting to me that we are using on the first container definition here.

Reply:
I tried to explain in the docs that the first container is going to be assumed is your main entry-point container

Reply:
Replaced with `primary_container_definition` and `extra_container_definitions` variables